packages/build-info/src/context.ts

```diff
@@ -23,7 +23,7 @@ export type ContextOptions = {
 
 export type Context = {
   projectDir: string
-  rootDir: string
+  rootDir?: string
```

Comment:
The root dir must not be set (could be only `projectDir` as well in the regular case of being not a mono repository)